.editorconfig

```diff
@@ -1,13 +1,18 @@
-# Editor configuration, see https://editorconfig.org
```

Comment:
I moved this from `./client` to the repo root to set consistent formatting configuration for all files in the repo.